tests/testthat/test-contribution_matrix.R

```diff
@@ -0,0 +1,244 @@
+
+test_that("CreateVMatrix() works for binary outcomes", {
+  data <- data.frame(Study = c("A", "A", "B", "B", "C", "C", "C", "D", "D"),
```

Comment:
This data frame is used in multiple test cases, could it be extracted into a `.csv` file and read in at the beginning of each case? That way any future changes to the data set can be done in one central location.

Reply:
Is there a reason that the file with a covariate has `;` as the delimiter instead of `,`?

Reply:
It seems to be the default in Open Office and I forgot to change it. Changed.